### ISSUE TITLE
Add support for Android 4.0

### ISSUE THREAD

**Issue by March-happy** [vector-im/riot-android#2310]
Now Riot.im only works for Android 4.1 and plus while Signal supports Android 4.0. I believe adding support for Android 4.0 users count because not everyone can afford a new phone or tablet in developing countries, and we may still use an old device.

**Comment by floviolleau** [comment]
I agree to your comment. I have a tablet that I cannot upgrade to the latest android. Is it complicated to support Android 4.0?

**Comment by bmarty** [comment]
Usually we increase the `minSdkVersion` value, it's like entropy :). Today is is set to 16 in Riot, new application should set it to 21 according to https://twitter.com/minsdkversion...

In practical, supporting old versions increase developer effort and support...
